lib/android-helpers.js

```diff
@@ -464,6 +464,11 @@ helpers.pushSettingsApp = async function (adb, throwError = false) {
 
   await helpers.installHelperApp(adb, settingsApkPath, SETTINGS_HELPER_PKG_ID, 'Settings');
 
+  // Making sure granting android.permission.SET_ANIMATION_SCALE
+  // Read Read https://github.com/appium/appium/pull/11640#issuecomment-438260477
+  logger.info('Granting android.permission.SET_ANIMATION_SCALE');
+  await adb.grantPermission(SETTINGS_HELPER_PKG_ID, 'android.permission.SET_ANIMATION_SCALE');
```

Comment:
Probably, it makes sense to do this only if settings app was not running before to save time (after `await adb.processExists(SETTINGS_HELPER_PKG_ID)` condition).